openapi-cli/src/main/java/io/ballerina/openapi/generators/GeneratorUtils.java

```diff
@@ -91,16 +91,7 @@
 import static io.ballerina.compiler.syntax.tree.SyntaxKind.SEMICOLON_TOKEN;
 import static io.ballerina.compiler.syntax.tree.SyntaxKind.STRING_KEYWORD;
 import static io.ballerina.openapi.cmd.OpenApiMesseges.BAL_KEYWORDS;
-import static io.ballerina.openapi.generators.GeneratorConstants.BALLERINA;
-import static io.ballerina.openapi.generators.GeneratorConstants.DELETE;
-import static io.ballerina.openapi.generators.GeneratorConstants.EXPLODE;
-import static io.ballerina.openapi.generators.GeneratorConstants.GET;
-import static io.ballerina.openapi.generators.GeneratorConstants.HEAD;
-import static io.ballerina.openapi.generators.GeneratorConstants.OPTIONS;
-import static io.ballerina.openapi.generators.GeneratorConstants.PATCH;
-import static io.ballerina.openapi.generators.GeneratorConstants.PUT;
-import static io.ballerina.openapi.generators.GeneratorConstants.STYLE;
-import static io.ballerina.openapi.generators.GeneratorConstants.TRACE;
+import static io.ballerina.openapi.generators.GeneratorConstants.*;
```

Comment:
We need to import necessary constants one by one.

Reply:
Fixed